### ISSUE TITLE
Input samples

### ISSUE THREAD

**Issue by tolstenko** [KatzuoOgust/Untech.DataProcessing#1]
Can you share some input samples?

**Comment by KatzuoOgust** [comment]
Yes, sure. I'll try to find and provide some input samples during this week.

**Comment by tolstenko** [comment]
Thank you

**Comment by KatzuoOgust** [comment]
[BTSync](https://link.getsync.com/#f=Carcam&sz=4E8&t=1&s=H6X6AO2JUT7QZMUSNAJZS57S6LXXEHRM&i=CHP5GCSDMXAYKDFGKLP2ENPXXTSA73YLD&e=1470767610&v=2.3)
This link will be valid next 7 days.

**Comment by l15423286y** [comment]
I want to test hyperspectral images in raw format.Can you share some input samples?
